community/planning/libsawtooth_receipt_store.md

```diff
@@ -0,0 +1,136 @@
+# Libsawtooth Receipt Store
+<!--
+  Copyright 2018-2021 Cargill Incorporated
+  Licensed under Creative Commons Attribution 4.0 International License
+  https://creativecommons.org/licenses/by/4.0/
+-->
+
+## Summary
+[summary]: #summary
+
+This document discusses the current purpose and design of the receipt store in
+libsawtooth as well as proposes a new `ReceiptStore` trait.
+
+### Existing Receipt Store
+
+The purpose of the receipt store in libsawtooth is to store transaction receipts.
+Transaction receipts contain information related to the execution of a
+transaction. The receipt store allows for information related to the execution
+of a transaction that should not be stored in state to be saved to an underlying
+database.
+
+The `TransactionReceiptStore` is a struct that is a wrapper around an
+`OrderedStore`. The `OrderedStore` is a key value store indexed by a type. The
+receipt store uses transaction id as the key, transaction receipt as the value
+and is indexed by u64s. The `OrderedStore` trait is implemented for various
+backends including lmdb, redis and btree. The `OrderedStore` maintains the order
```

Comment:
Nitpick: Everywhere you say lmdb it should be LMDB (its done in some places) also id should be ID

Reply:
Updated